MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -8541,4 +8542,22 @@ public boolean checkYearlyRetirements() {
         }
         return false;
     }
+
+	/**
+	 * Returns the type of rating method as selected in the Campaign Options dialog. Lazy-loaded for performance.
+	 */
+	public IUnitRating getUnitRating() {
+		if(unitRating == null) {
+			UnitRatingMethod method = getCampaignOptions().getUnitRatingMethod();
+			
+		    if (method.equals(UnitRatingMethod.FLD_MAN_MERCS_REV)) {
```

Comment:
This should be flipped around against the off-chance that _method_ somehow ends up being null.